### DISCUSSION TITLE
使用‘时间维度聚合’查询时，返回的ts的含义是什么？

### DISCUSSION THREAD

**Discussion by Luokun2016** [taosdata/TDengine#4630] [General]
使用`时间维度聚合`查询时，返回的时间点含义是什么？
  如下图，数据库中的数据从 `2020-12-18 07:57:01.000`才有数据，
 时间维度聚合后时间就从`2020-12-18 07:55:00.000`开始的
![td1](https://user-images.githubusercontent.com/24650960/102595690-fd6f8500-4152-11eb-9d70-d6aa86aadc25.png)
![td2](https://user-images.githubusercontent.com/24650960/102595697-006a7580-4153-11eb-904b-e3f47b1f57fa.png)

1. 
`select * from fcloudqa.tb4556512529240555520 where ts > '2020-12-18 00:00:00' and ts < '2020-12-18 23:59:59' `
```
{
    "status": "succ",
    "head": [
        "ts",
        "val"
    ],
    "data": [
        [
            "2020-12-18 07:57:01.000",
            0.000000000
        ],
        [
            "2020-12-18 07:57:02.000",
            0.000000000
        ],
        [
            "2020-12-18 07:57:03.000",
            0.000000000
        ],
        [
            "2020-12-18 07:57:04.000",
            0.000000000
        ],
        [
```

2.
`select avg(val) from fcloudqa.tb4556512529240555520 where ts > '2020-12-18 00:00:00' and ts < '2020-12-18 23:59:59' interval(5m)`

```
{
    "status": "succ",
    "head": [
        "ts",
        "avg(val)"
    ],
    "data": [
        [
            "2020-12-18 07:55:00.000",
            0.000000000
        ],
        [
            "2020-12-18 08:00:00.000",
            0.000000000
        ],
        [
            "2020-12-18 08:05:00.000",
            0.000000000
        ],
        [
            "2020-12-18 08:10:00.000",
            0.000000000
        ],
```

**Comment by sangshuduo** [comment]
因为用了 interval ，结果返回的 ts 是时间窗口的起始时间
